modules/grpc/common/filterx/protobuf-field-converter.cpp

```diff
@@ -397,6 +501,20 @@ class BytesFieldConverter : public ProtobufFieldConverter
       }
     return true;
   }
+
+  bool add(Message *message, ProtoReflectors reflectors, FilterXObject *object)
+  {
+    try
+      {
+        reflectors.reflection->AddString(message, reflectors.fieldDescriptor, this->extract(object));
```

Comment:
It was an older decision, but is it safe to use string for bytes?

Reply:
There is not bytes API, string is used for that.

The API does not explicitly state this, though, but I cannot imagine anything else being used for that:
https://protobuf.dev/reference/cpp/api-docs/google.protobuf.message/